src/app/tasks/task-comments-viewer/task-comments-viewer.component.html

```diff
@@ -0,0 +1,132 @@
+        </p>
+        <div
+          class="comment"
+          [ngClass]="{
+            'comment-by-other-user': !comment.author_is_me,
+            'new-comment': comment.is_new
+          }"
+        >
+          <user-icon
+            *ngIf="
+              comment.should_show_avatar && shouldShowAuthorIcon(comment.type)
+            "
+            size="50"
+            user="comment.author"
+            [ngClass]="{ 'hidden-xs hidden-sm': comment.author_is_me }"
+          >
+          </user-icon>
+
+          <div class="comment-container" [ngSwitch]="comment.type">
+            <div *ngSwitchCase="status" class="comment-status">
+              <hr class="hr-text" data-content="{{ comment.comment }}" />
+            </div>
+
+            <div class="bubble" [ngClass]="commentClasses(comment)">
+              <a
+                (click)="scrollToComment(comment.original_comment.id)"
+                class="original-comment"
+                *ngIf="!!comment.reply_to_id"
+              >
+                <p class="reply-text">
+                  <i class="fa fa-reply"></i>
+                  {{
+                    comment.original_comment
+                      ? comment.original_comment.comment
+                      : "comment removed"
+                  }}
+                </p>
+              </a>
+              <div *ngSwitchCase="'extension'" class="comment-extension">
+                <extension-comment [task]="task" [comment]="comment">
+                </extension-comment>
+              </div>
+
+              <div *ngSwitchCase="'text'" class="comment-text">
+                <div
+                  class="markdown-to-html"
+                  [innerHTML]="comment.comment"
+                >
+              </div>
+            </div>
+            <!--TODO: pipe to markdown -->
+
+              <div *ngSwitchCase="'audio'" class="comment-audio">
+                <audio-player></audio-player>
+              </div>
+
+              <div *ngSwitchCase="'discussion'" class="comment-discussion">
+                <intelligent-discussion-player
+                  [task]="task"
+                  [discussion]="comment"
+                ></intelligent-discussion-player>
+              </div>
+
+              <div *ngSwitchCase="'image'" class="comment-image">
+                <a (click)="openCommentsModal(comment)">
+                  <img
+                    [src]="getCommentAttachment(comment)"
+                    width="100%"
+                    height="100%"
+                  />
+                </a>
+              </div>
+
+              <div *ngSwitchCase="'pdf'" class="comment-pdf">
+                <a (click)="openCommentsModal(comment)">
+                  <p><i class="fa fa-file-pdf-o"></i> view pdf</p>
+                </a>
+              </div>
+            </div>
+            <div *ngIf="isBubbleComment(comment.type)" class="comment-overflow">
+              <comment-bubble-action
+                ng-show="hover"
```

Comment:
For this one , cant be [hidden]

Reply:
I'm not sure what you're referring to, this is outdated.